packages/ui/src/atoms/heading/index.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { Slot } from '@radix-ui/react-slot';
+import { cva, type VariantProps } from 'class-variance-authority';
+import { cn } from '@hypha-platform/ui-utils';
+
+const headingVariants = cva(
+  'font-family-inherit font-medium leading-tight m-0',
+  {
+    variants: {
+      size: {
+        1: 'text-[14px] leading-tight',  // h9
```

Comment:
We should probably define those at the `tailwind.config.js` to keep all configuration in one location